Code/Framework/AzCore/AzCore/Asset/AssetDataStream.cpp

```diff
@@ -22,6 +22,7 @@ namespace AZ::Data
     {
         struct AssetDataStreamPrivate
         {
+            AZ_CLASS_ALLOCATOR(AssetDataStreamPrivate, SystemAllocator, 0);
```

Comment:
Nit: I think that last parameter is no longer needed, I'm pretty sure it got deprecated a few weeks ago. Ideally, new instances of this macro getting added shouldn't use it.

Reply:
Maybe I could create another PR to remove the redundant third parameter referenced in all of code files.

Reply:
I've submitted #14530, please help review code.

Reply:
> Nit: I think that last parameter is no longer needed, I'm pretty sure it got deprecated a few weeks ago. Ideally, new instances of this macro getting added shouldn't use it.

done.
The last parameter has been removed.